jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/delegate/AbstractFieldDelegate.java

```diff
@@ -252,6 +254,12 @@ public void setConversionInvalid(boolean conversionInvalid) {
     }
 
     public void updateInvalidState() {
+        // skip initial validation when the form is opened
```

Comment:
It's not actually true. Validation is performed when the required state is changed. In particular, due to data binding. What if the user disabled the required indicator and then enabled it again? Let's handle required state changes in general.